pio/src/iSpindel.cpp

```diff
@@ -646,6 +646,22 @@ bool uploadData(uint8_t service)
 
     return sender.sendInfluxDB(myData.server, myData.port, myData.db, myData.name, myData.username, myData.password);
   }
+  if (service == DTInfluxDBHTTPS)
+  {
+    sender.add("tilt", Tilt);
```

Comment:
All the copy-paste started from this line can be omitted. `code`
`
#if API_INFLUXDB
  if (service == DTInfluxDB **|| service == DTInfluxDBHTTPS**)
  {
    sender.add("tilt", Tilt);
    sender.add("temperature", scaleTemperatureFromC(Temperatur, myData.tempscale));
    sender.add("temp_units", tempScaleLabel(myData.tempscale));
    sender.add("battery", Volt);
    sender.add("gravity", Gravity);
    sender.add("interval", myData.sleeptime);
    sender.add("RSSI", WiFi.RSSI());
    CONSOLELN(F("\ncalling InfluxDB"));
    CONSOLELN(String(F("Sending to db: ")) + myData.db + String(F(" w/ credentials: ")) + myData.username +
              String(F(":")) + myData.password);
   
**if (service == DTInfluxDB)
{
   return sender.sendInfluxDB(myData.server, myData.port, myData.db, myData.name, myData.username, myData.password);
}
else
{
    return sender.sendInfluxDBHTTPS(myData.server, myData.port, myData.db, myData.name, myData.username,
                                    myData.password);
}**
`

Reply:
Reimplemented to use a bool/checkbox, as suggested.